examples/epstein_civil_violence/epstein_civil_violence/agent.py

```diff
@@ -3,13 +3,23 @@
 import mesa
 
 
-class Citizen(mesa.Agent):
+class EpsteinAgent(mesa.experimental.cell_space.CellAgent):
+    def update_neighbors(self):
+        """
+        Look around and see who my neighbors are
+        """
+        self.neighborhood = self.cell.get_neighborhood(radius=self.vision)
+
+        self.neighbors = self.neighborhood.agents
+        self.empty_neighbors = [c for c in self.neighborhood if c.is_empty]
```

Comment:
Vision is something we might want to build in at some point. NetLogo has quite extensive support for it, for example.